astropy/modeling/tests/test_parameters.py

```diff
@@ -809,37 +809,25 @@ def test_bool(self):
         param = Parameter(name="test", default=1)
         assert param.value == 1
         assert np.all(param)
-        if param:
-            assert True
-        else:
-            assert False
+        assert not param
```

Comment:
```suggestion
        assert param
```
This should fix the failing CI. This is the test for truth so it's backwards to the rest of them.